binderhub/app.py

```diff
@@ -107,6 +107,17 @@ def _log_level(self):
         config=True
     )
 
+    news_message = Unicode(
```

Comment:
Could we call this "banner message" instead of "news message"? I feel like that's a bit more descriptive and it follows terminology that lots of other sites use (I'm thinking here what a person would google for to learn how to do this, "How to add a banner to my BinderHub" sounds more likely to me than "How to add a news message to my BinderHub").

Reply:
Good point! Done.